src/app/modules/gb-side-panel-module/accordion-components/gb-cohorts/gb-cohorts.component.ts

```diff
@@ -234,11 +234,9 @@ export class GbCohortsComponent implements OnInit {
     return this.cohortService.isCohortSubscriptionIncluded;
   }
 
-  get subjectAllCount(): string {
-    return FormatHelper.formatCountNumber(this.cohortService.allCounts.subjectCount);
-  }
-
-  get observationAllCount(): string {
-    return FormatHelper.formatCountNumber(this.cohortService.allCounts.observationCount);
+  get subjectCountText(): string {
+    const count = this.cohortService.allCounts.subjectCount;
+    const countString = FormatHelper.formatCountNumber(this.cohortService.allCounts.subjectCount)
```

Comment:
will simplify this to `const countString = FormatHelper.formatCountNumber(count)`